lib/integration/queryManager.js

```diff
@@ -0,0 +1,34 @@
+import { logMessage } from "../logger";
```

Comment:
Why is this a JS file here. If we're adding new things to the code base its a good opportunity to write them in TypeScript